src/analyze/analyze-security.c

```diff
@@ -17,7 +17,7 @@
 #include "path-util.h"
 #include "pretty-print.h"
 #if HAVE_SECCOMP
-#  include "seccomp-util.h"
+#        include "seccomp-util.h"
```

Comment:
We use two-space indentation for `#if`s everywhere. Let's not change thsi.

Reply:
I can do that. clang-format is not capable of that. but i can implement this in the tools file.